net-misc/rygel/rygel-0.40.4.ebuild

```diff
@@ -36,13 +37,15 @@ DEPEND="
 		dev-libs/libunistring:=
 		x11-libs/gdk-pixbuf:2
 	)
-	tracker? ( app-misc/tracker:3= )
+	tracker? ( app-misc/tracker:3 )
 	transcode? (
 		media-libs/gst-plugins-bad:1.0
 		media-plugins/gst-plugins-twolame:1.0
 		media-plugins/gst-plugins-libav:1.0
 	)
 	gtk? ( >=x11-libs/gtk+-3.22:3 )
+
+	x11-libs/libX11
```

Comment:
Whaaaa?

The x11 dep was added in https://gitlab.gnome.org/GNOME/rygel/-/commit/18cd2cff6f64b7a54156ac97e5c4f50eb8fd8d67 AFAICT, which is not in v0.40.

In general, we shouldn't be modifying stable ebuilds, and we shouldn't be modifying other ebuilds in version bump commits.

Reply:
oh ffs I accidentally added this in, then I removed this, then did the rebase from your branch and added it again this way.  -.-